docgen/src/guides/usage.md.hbs

```diff
@@ -82,3 +82,13 @@ const search = instantsearch({ ... });
 search.addWidget(searchBox({ ... }));
 search.addWidget(connectSearchBox(function() { ... })({ ... }))
 ```
+
+## Browser support
+
+We support the last **two versions of major browsers** (Chrome, Edge, Firefox, Safari).
```

Comment:
"last" should also probably be `strong` → `**last two versions of major browsers**`.